core-bundle/src/Repository/CronJobRepository.php

```diff
@@ -14,8 +14,8 @@
 
 use Contao\CoreBundle\Entity\CronJob;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
-use Doctrine\Common\Persistence\ManagerRegistry;
 use Doctrine\DBAL\Connection;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
```

Comment:
How is that better than using `Doctrine\Persistence\ManagerRegistry`?

Reply:
Because it won't work over all versions (see 80de19a and c4612fe) otherwise.

Reply:
The functional tests on `prefer-lowest` do fail: https://github.com/contao/contao/pull/2321/checks?check_run_id=1301899600

Reply:
I see. 👍🏻